src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Renderer/DrawingViewRenderer.android.cs

```diff
@@ -168,7 +158,7 @@ public override bool OnInterceptTouchEvent(MotionEvent ev)
 			return base.OnInterceptTouchEvent(ev);
 		}
 
-		IList<Point> NormalizePoints(IList<Point> points)
+		IList<Point> NormalizePoints(IEnumerable<Point> points)
```

Comment:
Just out of curiosity, why did you change the parameter type?

Reply:
I firstly added IList in the previous commit. we can use a more abstract type here (we do not need Add,Remove behavior from IList)

Reply:
thanks for the answer.